### ISSUE TITLE
Cannot achieve equal bounce angle with circle and polygon shape

### ISSUE THREAD

**Issue by wojciak** [schteppe/p2.js#156]
Hello,

TL;DR: I cannot seem to achieve this http://cimss.ssec.wisc.edu/wxwise/class/reflect.gif

I'm building a 8-ball pool game and I've run into a problem:
When I hit the ball against the bound of the pool table, the bounce angle is getting "damped-out", so for example when I hit the bound of the table at a 45degree angle the ball bounces off at a 40degree angle and after few bounces back and forth it just moves in a straight line.

The ball's body is a simple circle to which I apply force[0] & force[1]. The table bound is a static body.fromPolygon.

P.s. when I play with angularForce I can make the ball behave in odd ways when bouncing. Can I achieve equal bounce with this property?

**Comment by schteppe** [comment]
Hi,

Do you get this behavior with other shapes too? Like Rectangle?

Try turning off friction. Better? If not, try using the same settings as the leftmost ball in the restitution demo: http://schteppe.github.io/p2.js/demos/restitution.html

**Comment by wojciak** [comment]
Thanks for the hints. Setting friction to 0 and adding fixedRotation did the trick - thank you again! :+1: 

Closing.
